docs/getting-started.md

```diff
@@ -75,6 +75,17 @@ The primary purpose of the variable is to allow DigitalOcean customers to easily
 
 When a cluster is created in a non-default VPC for the region, the environment variable `DO_CLUSTER_VPC_ID` must be specified or Load Balancer creation for services will fail.
 
+### Load-balancer ID annotations
+
+`digitalocean-cloud-controller-manager` attaches the UUID of load-balancers to the corresponding Service objects (given they are of type `LoadBalancer`) using the `service.k8s.digitalocean.com/load-balancer-id` annotation. This serves two purposes:
+
+1. To support load-balancer renames without impacting the correct working of `digitalocean-cloud-controller-manager`.
+2. To efficiently look up load-balancer resources in the DigitalOcean API.
+
+The annotation is added for both newly created and existing load-balancers. However, existing load-balancers lacking the annotation yet can only be associated correctly if they were not renamed before. Otherwise, the renamed load-balancer will be considered missing and a new one will be created instead, leaving the old one dangling.
+
+It is also possible to start owning an unmanaged load-balancer by adding the annotation manually prior to setting the Service type to `LoadBalancer`. Note however every load-balancer should be owned by exactly one `digitalocean-cloud-controller-manager` instance only to prevent concurrent and presumably conflicting modifications.
```

Comment:
> You can import an existing Load Balancer by creating the `Service` annotated with the ID of the Load Balancer. Don’t do this while it’s being managed by another cluster, or the `digitalocean-cloud-controller-managers` will make conflicting modifications to the Load Balancer.

Reply:
I went with your suggestion with two exceptions:

- I think "owning" is clearer than "importing"
- You can also own/import a load-balancer after the Service has been created as long as the type isn't set to `LoadBalancer` yet. I think the new wording makes this easy enough to understand now.

Reply:
Maybe we can just call it ‘managing’ instead of owning, because that’s what you call it in the next sentence too.

> You can also own/import a load-balancer after the Service has been created as long as the type isn't set to LoadBalancer yet

I think the way you’re wording this is confusing, because it makes it sounds like you need to:

1. Create a Service
2. Set the annotation
3. Change the type to LoadBalancer

While this is possible, it’s a very roundabout way of doing it. You can just create a service of type LoadBalancer with the annotation in one go. Perhaps you could say ‘You [...] manage an existing load-balancer by creating a LoadBalancer Service annotated with the UUID of the load-balancer.’

Also `or otherwise` is a tautology.

Reply:
Adopted your suggestion and rephrased the second sentence to be less tautological.

Reply:
Just dropped "otherwise" in the last sentence.